src/test/java/supertracker/storage/FileManagerTest.java

```diff
@@ -0,0 +1,55 @@
+package supertracker.storage;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import supertracker.command.NewCommand;
+import supertracker.item.Inventory;
+import supertracker.item.Item;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class FileManagerTest {
+    private static final String INVALID_EX_DATE = "01-01-99999";
+    private static final DateTimeFormatter INVALID_EX_DATE_FORMAT = DateTimeFormatter.ofPattern("dd-MM-yyyyy");
+    private static final LocalDate UNDEFINED_DATE = LocalDate.parse(INVALID_EX_DATE, INVALID_EX_DATE_FORMAT);
+    private static final LocalDate CURR_DATE = LocalDate.now();
+    @BeforeAll
```

Comment:
```suggestion
    private static final LocalDate CURR_DATE = LocalDate.now();

    @BeforeAll
```
Perhaps add some spacing here